modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -125,39 +127,74 @@ DEFAULT pxeboot
+`coreos.inst.ignition_url` parameter value is the location of the bootstrap
+Ignition config file on your HTTP server.
 
 ifndef::only-pxe[]
 ** For iPXE:
 +
 ----
-kernel  http://<HTTP_server>/rhcos-<version>-installer-kernel-<architecture> ip=dhcp rd.neednet=1 initrd=http://<HTTP_server>/rhcos-<version>-installer-initramfs.<architecture>.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-metal.<arhcitectutre>.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <1> <2>
-initrd http://<HTTP_server>/rhcos-<version>-installer-initramfs.<architecture>.img <3>
+kernel  http://<HTTP_server>/rhcos-<version>-<architecture>-installer-kernel-<architecture> ip=dhcp rd.neednet=1 initrd=rhcos-<version>-<architecture>-installer-initramfs.<architecture>.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-<architecture>-metal.<architecture>.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <1> <2>
+initrd http://<HTTP_server>/rhcos-<version>-<architecture>-installer-initramfs.<architecture>.img <3>
 boot
 ----
 <1> Specify locations of the {op-system} files that you uploaded to your
 HTTP server. The `kernel` parameter value is the location of the `kernel` file,
-the `initrd` parameter value is the location of the `initramfs` file,
-the `coreos.inst.image_url` parameter value is the location of the compressed
-metal RAW image, and the `coreos.inst.ignition_url` parameter value is the
-location of the bootstrap Ignition config file.
+the `initrd` parameter value references the name of the `initramfs` file that is
+supplied on the `initrd` line below, the `coreos.inst.image_url` parameter value
+is the location of the compressed metal RAW image, and the `coreos.inst.ignition_url`
+parameter value is the location of the bootstrap Ignition config file.
 <2> If you use multiple NICs, specify a single interface in the `ip` option.
 For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 <3> Specify the location of the `initramfs` file that you uploaded to your HTTP
 server.
 endif::only-pxe[]
 
+. If you use UEFI, perform the following actions:
+.. Provide the EFI binaries and `grub.cfg` file that are required for booting the system. You need the `shim.efi` binary and the `grubx64.efi` binary.
+
+** Extract the necessary EFI binaries by mounting the {op-system} ISO on
+your host, copying the `images/efiboot.img` file to your host, then mounting `efiboot.img` on your host. From the `efiboot.img` mount point, you then copy the `EFI/redhat/shimx64.efi` and
+`EFI/redhat/grubx64.efi` files to your TFTP server.
++
+----
+# mkdir -p /mnt/{iso,efiboot}
+# mount -o loop rhcos-installer.x86_64.iso /mnt/iso
+# cp /mnt/iso/images/efiboot.img .
+# mount -o loop  -t vfat efiboot.img /mnt/efiboot
```

Comment:
We could simplify this by skipping the `cp` and doing `mount -o loop,ro` directly from `/mnt/iso/images/efiboot.img`.  The summary description above would need to be updated also.

Reply:
Good suggestion!